Runtime_Support/dot_net_core.md

```diff
@@ -1,25 +1,35 @@
-# .NET Core on App Service
+# .NET on App Service
 
-## .NET Core Update Policy
+## .NET Update Policy
 
-App Service upgrades the underlying .NET Core runtime and SDK of your application as part of the regular platform updates. As a result of this update process, your application will be automatically updated to the latest patch version available in the platform for the configured minor version of your app.
+App Service upgrades the underlying .NET runtime and SDK of your application as part of the regular platform updates. As a result of this update, your application will be automatically updated to the latest patch version available in the platform for the configured minor version of your app.
 
 ### End of Life
 
 Once a version of .NET Core has reached it's end of life (EOL) it will no longer be available from Runtime Stack selection dropdown.
 
 Existing applications configured to target a runtime version that has reached EOL should not be affected.
 
+## .NET 5 Early Access limitations
+
+During the early access period, .NET 5 apps will be subject to the **Early Access** limitations listed in the [early access support document](./early_access.md).
+
+[Application Insights](https://azure.microsoft.com/services/monitor) is currently not supported for .NET 5 apps, we are working with the Application Insights team to resolve this.
+
+Visual Studio and Visual Studio for Mac will not support creating new .NET 5 Apps on App Service through the IDE at launch. As a workaround .NET 5 Apps can be created from the Azure Portal or Azure CLI.
+Publishing content to a .NET 5 works as expected. There is schedule update  that will remove this limitation.
```

Comment:
FYI @bradygaster let me know if you want to change anything here

Reply:
you have an extra space between update and that

Reply:
fixed